install_config/install/advanced_install.adoc

```diff
@@ -1349,9 +1346,44 @@ set for a region called `primary` and a zone called `east`:
 node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
 ----
 
+Starting in {product-title} 3.9, masters are now marked as schedulable nodes by
+default. As a result, the default node selector (defined in the master
+configuration file's `projectConfig.defaultNodeSelector` field to determine
+which node that projects will use by default when placing pods, and previously
+left blank by default) is now set by default during cluster installations. It is
+set to `node-role.kubernetes.io/compute=true` unless overridden using the
+`osm_default_node_selector` Ansible variable.
+
+In addition, as long as `osm_default_node_selector` is left unset, any
```

Comment:
Per QE's testing in https://bugzilla.redhat.com/show_bug.cgi?id=1548641#c18, whatever osm_default_node_selector is set or NOT, this automatic node labeling always occurs. Master nodes are labeled with node-role.kubernetes.io/master=true, non-master, non-dedicated infrastructure nodes hosts (nodes with a region=infra label) are labeled with node-role.kubernetes.io/compute=true.